controllers/grid/issues/form/IssueGalleyForm.php

```diff
@@ -218,6 +218,7 @@ public function execute(...$functionArgs)
 
             if ($this->getData('label') == null) {
                 // Generate initial label based on file type
+                $fileType = $issueGalleyFile->getContentType();
```

Comment:
@asmecher The `$fileType` is undefined at this point, so I guessed it should be set this way.

Reply:
It looks fine at a glance; is it tested to work when you create an issue galley?

Reply:
I watched around and removed this whole piece (the label field is required).